great_expectations/dataset/sparkdf_dataset.py

```diff
@@ -179,7 +179,7 @@ def __init__(self, spark_df, *args, **kwargs):
     def head(self, n=5):
```

Comment:
All changes in this file are just merge conflict resolutions between feature/ntiles and feature/asana branches